app/frontend/tsconfig.json

```diff
@@ -18,7 +18,7 @@
       }
     ],
     "paths": {
-      "@/*": ["./*"]
+      "@/*": ["./src/*"]
```

Comment:
**Refrain from using the src directory**

While it is a good practice in other projects. We will not be using the src directory for our use case. Please init the project without the src directory.